src/applications/claims-status/tests/components/claim-document-request-pages/DefaultPage.unit.spec.jsx

```diff
@@ -25,7 +25,7 @@ describe('<DefaultPage>', () => {
     uploading: false,
   };
 
-  it('should render component when status is NEEDED_FROM_YOU', () => {
+  it.skip('should render component when status is NEEDED_FROM_YOU', () => {
```

Comment:
Why are we skipping this?

Reply:
Hi @jerekshoe I left a comment above about this with a screenshot. I'm not sure why it's failing but the commit this PR is reverting introduced unit test failures in main so I'm just trying to resolve that. Any idea why this unit tests is failing though?

Relevant Slack thread: https://dsva.slack.com/archives/CBU0KDSB1/p1737501402247669

Reply:
Yeah sorry I saw your comment after I added mine, this is something we need to resolve on our side